src/course/views.py

```diff
@@ -37,8 +37,8 @@ class CourseMasterView(MethodView):
 
     def get(self, courseID):
         course = models.Course.query.filter_by(id=int(courseID) - 1000).first()
-        author = models.User.query.filter_by(id=course.teacher_id).first()
-        if course in current_user.courses or course.teacher_id == current_user.id:
+        author = models.User.query.filter_by(id=course.teacher_id).first()     
+        if course in current_user.courses or course.teacher_id == current_user.id or ","+str(current_user.id)+"," in course.secondaryTeachers:
```

Comment:
in the future split this up across multiple lines and put spaces between all operators